modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer.js

```diff
@@ -29,8 +29,10 @@ const DEFAULT_GAMMA = 0.2;
 const DEFAULT_BUFFER = 192.0 / 256;
 
 const defaultProps = {
+  getLineLength: {type: 'accessor', value: x => x.lineLength || 0},
   getShiftInQueue: {type: 'accessor', value: x => x.shift || 0},
```

Comment:
Looks like the format of the return value has changed?

Reply:
What do you mean by saying `the format of the return value has changed`?

Reply:
When you call `getShiftInQueue` you are expecting it to return an array. And the default value set here is a number.